### PR TITLE
Improve the dev feed section

### PR DESCRIPTION
**Pull request by jongio**
We recently had a customer trying to consume the dev feed packages, but struggled with the instructions as they are today.  I took a stab at improving them.

# All SDK Contribution checklist:

This checklist is used to make sure that common guidelines for a pull request are followed.
- [ x] **Please open PR in `Draft` mode if it is:**
	- Work in progress or not intended to be merged.
	- Encountering multiple pipeline failures and working on fixes.
- [ x] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [ x] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/master/CONTRIBUTING.md).**
- [ x] **The pull request does not introduce [breaking changes](https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/breaking-change-rules.md).**

### [General Guidelines and Best Practices](https://github.com/Azure/azure-sdk-for-net/blob/master/CONTRIBUTING.md#general-guidelines)
- [ x] Title of the pull request is clear and informative.
- [ x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/master/CONTRIBUTING.md#testing-guidelines)
- [ x] Pull request includes test coverage for the included changes.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/master/CONTRIBUTING.md#sdk-generation-guidelines)
- [x ] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as the commitid of your swagger spec or link to the swagger spec, used to generate the code. (Track 2 only)
- [ x] The `*.csproj` and `AssemblyInfo.cs` files have been updated with the new version of the SDK. Please double check nuget.org current release version.

## Additional management plane SDK specific contribution checklist: 
Note: Only applies to `Microsoft.Azure.Management.[RP]` or `Azure.ResourceManager.[RP]`
 
- [ ] Include updated [management metadata](https://github.com/Azure/azure-sdk-for-net/tree/master/eng/mgmt/mgmtmetadata).
- [ ] Update AzureRP.props to add/remove version info to maintain up to date API versions.

### Management plane SDK Troubleshooting
- If this is very first SDK for a services and you are adding new service folders directly under /SDK, please add `new service` label and/or contact assigned reviewer.
- If the check fails at the `Verify Code Generation` step, please ensure:
	- Do not modify any code in generated folders.
	- Do not selectively include/remove generated files in the PR.
	- Do use `generate.ps1/cmd` to generate this PR instead of calling `autorest` directly.
	Please pay attention to the @microsoft.csharp version output after running generate.ps1. If it is lower than current released version (2.3.82), please run it again as it should pull down the latest version,

### Old outstanding PR cleanup
 Please note:
	If PRs (including draft) has been out for more than 60 days and there are no responses from our query or followups, they will be closed to maintain a concise list for our reviewers.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -251,46 +251,94 @@ Each library needs to provide a `ApiCompatVersion` property which is set to the
...
 <Project Sdk="Microsoft.NET.Sdk">
-  <PropertyGroup>
-    <OutputType>Exe</OutputType>
-    <TargetFramework>netcoreapp3.1</TargetFramework>
-  </PropertyGroup>
   <ItemGroup>
-    <PackageReference Include="Azure.Identity" Version="1.2.0-dev.*" />
+    <PackageReference Include="Azure.Data.Tables" Version="3.0.0-alpha.*"/>
   </ItemGroup>
+  <PropertyGroup>
```
Comment (weshaggard) [inline CONTRIBUTING.md:335]: I don't think we should use this type of reference it is highly recommended that folks use the Nuget.Config.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -251,46 +251,94 @@ Each library needs to provide a `ApiCompatVersion` property which is set to the
...
+## NuGet Package Dev Feed
 
-Follow instructions provided [here](https://docs.microsoft.com/nuget/consume-packages/install-use-packages-visual-studio#package-sources) and use `https://pkgs.dev.azure.com/azure-sdk/public/_packaging/azure-sdk-for-net/nuget/v3/index.json` as the source.
+The Azure SDK for .NET releases packages daily from our CI pipeline to our NuGet package dev feed to help developers use and test new libraries before they are officially released to NuGet.
 
-You can also achieve this from the command line.
+**Dev Feed Package Browser**:
 
-```nuget.exe sources add -Name "Azure SDK for Net Dev Feed" -Source "https://pkgs.dev.azure.com/azure-sdk/public/_packaging/azure-sdk-for-net/nuget/v3/index.json"```
+1. https://aka.ms/azsdk/net/feed
```
Comment (weshaggard) [inline CONTRIBUTING.md:260]: Do you really want to include both links given you made aka links?

File: CONTRIBUTING.md (hunk 1 above)
Comment (jongio) [reply CONTRIBUTING.md:335]: Can you tell me more about that?  I haven't heard that before.

File: CONTRIBUTING.md (hunk 2 above)
Comment (jongio) [reply CONTRIBUTING.md:260]: I was thinking that some users would want short and others would want to see what the URL is.

Comment (weshaggard) [reply CONTRIBUTING.md:260]: Given that most people will simply copy and paste this link I wouldn't even suggest we hide it behind an aka link. Having the full link allows folks to better identify where it is coming from.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -251,46 +251,78 @@ Each library needs to provide a `ApiCompatVersion` property which is set to the
...
 ```
 
-You can then consume packages from this package source, remember to check the [Include prerelease](https://docs.microsoft.com/nuget/create-packages/prerelease-packages#installing-and-updating-pre-release-packages) box in Visual Studio when searching for the packages. 
+> You can place a NuGet.Config file in the root of your solution. Projects within the solution will use the feed defined in that file.
+
+### 2. Find NuGet Package
+
+You can use the following options to find the available dev feed packages:
+
+1. Search the Azure SDK for .NET Dev Feed: https://aka.ms/azsdk/net/feed
```
Comment (weshaggard) [inline CONTRIBUTING.md:307]:
```suggestion
1. Search the Azure SDK for .NET Dev Feed: https://dev.azure.com/azure-sdk/public/_packaging?_a=feed&feed=azure-sdk-for-net
```

**Review by weshaggard — APPROVED** [review]

**Pull request merged**